src/controllers/dashboard/general.js

```diff
@@ -36,6 +37,7 @@ import alert from '../../components/alert';
             config.MetadataPath = $('#txtMetadataPath', form).val();
             config.MetadataNetworkPath = $('#txtMetadataNetworkPath', form).val();
             config.QuickConnectAvailable = form.querySelector('#chkQuickConnectAvailable').checked;
+            config.ParallelImageEncodingLimit = parseInt(form.querySelector('#txtParallelImageEncodingLimit').value || '1');
```

Comment:
A radix should be specified when calling `parseInt` to avoid some browser parsing number strings starting with "0" being parsed as a hexadecimal.

```suggestion
            config.ParallelImageEncodingLimit = parseInt(form.querySelector('#txtParallelImageEncodingLimit').value || '1', 10);
```

Reply:
Fixed. Thanks :)